src/tdom/__init__.py

```diff
@@ -6,12 +6,18 @@
 _parsed = {}
 _listeners = []
 
-T = (t'').__class__
-
+# T = (t'').__class__
+# Let's just use the imported class
+try:
+  # Temporary hack because pytest-playwright has to run under 3.13
+  # in CI due to greenlet not compiling there under 3.14.
+  from string.templatelib import Template
+except ImportError:
+  pass
```

Comment:
can we put `T = (t'').__class__` in here instead? I am sure MicroPython will fail that `from` import

Reply:
Sure, but it will only fail because MicroPython doesn't support t-strings at all. When it supports t-strings, that import will have to work.